optaweb-vehicle-routing-frontend/cypress/integration/fromLocationsToRoute.js

```diff
@@ -37,15 +37,13 @@ describe('Locations can be added and route is computed', () => {
   /**
    * Waits for a websocket connection to be established.
    */
-  const waitForWebSocketToConnect = () => {
-    cy.server();
-    cy.route('/vrp-websocket/*').as('vrp');
-    cy.visit('/demo');
-    cy.wait('@vrp');
+  const visitDemo = () => {
+    cy.visit('/');
+    cy.get(':nth-child(4) > .pf-c-nav__link').click();
```

Comment:
This will break once we reorder menu items.

Reply:
I know, unfortunately, since the Link is not our component, I cannot put there `data-cy` attribute. Possibly I can wrap each link by `<div>` and put the attribute there.

Reply:
hmm, likely there will be a better way, let me try.